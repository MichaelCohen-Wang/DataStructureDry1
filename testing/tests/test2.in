get_cannons 468
add_ship 44 4
get_cannons 44
add_ship 408 14
add_pirate 301 512 11
add_ship 273 6
get_cannons 273
remove_pirate 301