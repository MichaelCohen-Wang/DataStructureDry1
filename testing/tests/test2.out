get_cannons: FAILURE
add_ship: SUCCESS
get_cannons: SUCCESS, 4
add_ship: SUCCESS
add_pirate: FAILURE
add_ship: SUCCESS
get_cannons: SUCCESS, 6
remove_pirate: FAILURE
